src/senaite/core/browser/controlpanel/samplingdeviations/view.py

```diff
@@ -0,0 +1,125 @@
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+import collections
+
+from bika.lims import api
+from bika.lims import senaiteMessageFactory as _
+from bika.lims.utils import get_link_for
+from senaite.core.i18n import translate
+from senaite.app.listing import ListingView
+from senaite.core.catalog import SETUP_CATALOG
+from senaite.core.permissions import AddSamplingDeviation
+
+
+class SamplingDeviationsView(ListingView):
+
+    def __init__(self, context, request):
+        super(SamplingDeviationsView, self).__init__(context, request)
+
+        self.catalog = SETUP_CATALOG
+        self.contentFilter = {
+            "portal_type": "SamplingDeviation",
+            "sort_on": "sortable_title",
+            "sort_order": "ascending",
+            "path": {
+                "query": api.get_path(self.context),
+                "depth": 1,
+            },
+        }
+
+        self.context_actions = {
+            _("listing_samplingdeviations_action_add", default="Add"): {
+                "url": "++add++SamplingDeviation",
+                "permission": AddSamplingDeviation,
+                "icon": "senaite_theme/icon/plus"
+            }
+        }
+
+        self.title = translate(_(
+            "listing_samplingdeviations_title",
+            default="Sampling Deviations")
+        )
+        self.icon = api.get_icon("SamplingDeviations", html_tag=False)
+        self.show_select_column = True
+
+        self.columns = collections.OrderedDict((
+            ("Title", {
+                "title": _(
+                    "listing_samplingdeviations_column_title",
+                    default="Sampling Deviation",
+                ),
+                "index": "sortable_title",
+            }),
+            ("Description", {
+                "title": _(
+                    "listing_samplingdeviations_column_description",
+                    default="Description",
+                ),
+                "index": "Description",
+                "toggle": True,
+            }),
+        ))
+
+        self.review_states = [
+            {
+                "id": "default",
+                "title": _(
+                    "listing_samplingdeviations_state_all",
+                    default="All",
+                ),
+                "contentFilter": {},
+                "transitions": [{"id": "empty"}, ],
+                "columns": self.columns.keys(),
+            }, {
+                "id": "active",
+                "title": _(
+                    "listing_samplingdeviations_state_active",
+                    default="Active",
+                ),
+                "contentFilter": {"is_active": True},
+                "transitions": [{"id": "deactivate"}, ],
```

Comment:
Remove this `transitions` attribute. Listing displays the available transitions for the selected object automatically based on the workflow(s) the object is bound to.